lib/hawkular/alerts/alerts_api.rb

```diff
@@ -20,6 +20,13 @@ def initialize(entrypoint = 'http://localhost:8080/hawkular/alerts', credentials
       super(entrypoint, credentials, options)
     end
 
+    # Return version and status information for the used version of Hawkular-Alerting
+    # @return [Hash{String=>String}]
+    #         ('Implementation-Version', 'Built-From-Git-SHA1', 'status')
+    def fetch_version_and_status
```

Comment:
Yes and no.
Yes, at it is not covered in alerts subclient how to fetch the version.
No, as it is already part of the public api for metrics subclient:
https://github.com/hawkular/hawkular-client-ruby/blob/master/lib/hawkular/metrics/metric_api.rb#L18
So, it has not sense to have different names for the same function in different subclients.
And this PR has not intention to review the API in all places where this method is used.
As I commented, this PR is just to complete to add this feature for future uses.